quickwit/quickwit-config/src/index_config/mod.rs

```diff
@@ -112,13 +117,16 @@ impl Default for IndexingResources {
     }
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, utoipa::ToSchema)]
 #[serde(deny_unknown_fields)]
 pub struct IndexingSettings {
+    #[schema(default = json!(60))]
```

Comment:
no need to use `json!` macro, just putting a int is fine.

Reply:
I think when I tested this, utoipa converts this to a litteral string in the example, which is why it's a json!

Reply:
I checked it in swagger UI (served by warp so that I can easily check the result), and it's working. I did `#[schema(default = 8)]` for `docstore_compression_level`.

![Capture d’écran 2022-12-26 à 21 52 18](https://user-images.githubusercontent.com/653704/209582771-f2f49fbc-2ebe-4799-8d18-d87391e90369.png)